### PR TITLE
feat: navigation

### PR DESCRIPTION
**Pull request by anttio**
## 🖼 Context
image

To scale the documentation site, there is a need for extra level of navigation.

## 🚀 Changes
- Added responsive header with navigation
- Sidebar navigation now done with [Eleventy navigation plugin](https://www.11ty.dev/docs/plugins/navigation/)

## 🤔 Considerations

### CODE REVIEW

**Comment by gusso** [comment]
Screenshot 2020-10-07 at 1 43 55 pm
I think it'd be nice if the left side of the sidebar items aligned with the logo, and the selected status was hanging on the left. On mobile, the margin would push the navigation inwards and the alignment would be closer to what it is now.

**Comment by gusso** [comment]
Screenshot 2020-10-07 at 1 48 38 pm
Menu item links could use some horizontal padding as well as vertical, to increase the hit area.

**Comment by gusso** [comment]
Screenshot 2020-10-07 at 1 50 09 pm
The blue should look sharper if it doesn't have to fight the grey, so if you could overlap them I think it'll work better.

**Comment by gusso** [comment]
Screenshot 2020-10-07 at 1 54 28 pm
I think the relationship between the line and the menu item will be stronger if the proportion of the selected line is a bit smaller, like maybe a `size-24` height on the line.

**Comment by gusso** [comment]
There's no responsive panel for the component list yet, right?

**Comment by anttio** [comment]
> There's no responsive panel for the component list yet, right?

Not yet.

File: src/_assets/styles.css [hunk 1]
```diff
@@ -2,118 +2,248 @@
 @import '@transferwise/neptune-css/dist/css/neptune-core.css';
 @import '@transferwise/neptune-css/dist/css/table.css';
 
-/* Neptune Docs – Utilities */
+:root {
+  --body-text-color: #5d7079;
```
Comment (helendurranttw) [inline src/_assets/styles.css:6]: 😌

**Comment by helendurranttw** [comment]
The focus styling is getting cut off for some reason.

Screenshot 2020-10-08 at 13 54 29

**Comment by helendurranttw** [comment]
You don't have to address this in this PR, but general feedback - I wasn't expecting to find the statuses on the general Components page, though maybe this makes sense? 🤔 More of a content question I guess.

In any case, it would be good to include the status per page, as well.

**Comment by helendurranttw** [comment]
Let's do responsive now, or like neptune web, we might never do it 😁 could we just nest the whole sub-menu in the slide out menu?

**Comment by anttio** [comment]
> Let's do responsive now, or like neptune web, we might never do it 😁 could we just nest the whole sub-menu in the slide out menu?

Added the components list in the sub-menu for now. This needs more work when we start to scale the site with more content.

**Review by helendurranttw — APPROVED** [review]

**Pull request merged**